app/controllers/application_controller.rb

```diff
@@ -1,5 +1,19 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  protect_from_forgery with: :exception
+  # before_action :authenticate_user!
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :surname, :bio, :photo, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :surname, :bio, :photo, :email, :password, :current_password)
+    end
   end
+
+  # Catch all CanCan errors and alert the user of the exception
+  # rescue_from CanCan::AccessDenied do |exception|
+  #   redirect_to root_url, alert: exception.message
+  # end
```

Comment:
- [ ] To make your code professional, kindly remove commented codes from your file :)